src/tests/ftest/util/dmg_utils.py

```diff
@@ -1020,7 +1020,7 @@ def telemetry_metrics_query(self, host, metrics=None):
         return self._get_json_result(
             ("telemetry", "metrics", "query"), host=host, metrics=metrics)
 
-    def _parse_pool_list(self, key=None):
+    def _parse_pool_list(self, key=None, **kwargs):
```

Comment:
You might want to add description for **kwargs. Also for other get_pool_list_ methods below.